### PR TITLE
[Desktop] Rearrange pages to match design

### PR DESCRIPTION
**Pull request by git-f0x**
This matches the design in #439. Draft because desc for window-management is reported as not existing for some reason, so help on that would be welcome.
Screenshot without Window Management description:
screenshot-2024-08-02-19-23-07

Edit: Unrelated, but the non-scrolling page title disappears when the window is shrunk vertically, instead of staying and a scrollbar appearing, which results in the Desktop page with the extra entries opening without a title at the default window size.

### CODE REVIEW

File: i18n/en/cosmic_settings.ftl [hunk 1]
```diff
@@ -233,6 +224,20 @@ x-hours = { $number ->
     *[other] { $number } hours
 }
 
+## Desktop: Window management
+
+window-management = Window management
```
Comment (wash2) [inline i18n/en/cosmic_settings.ftl:229]: There is already a key for window-management on line 69, so you can add the new description there.

Comment (git-f0x) [reply i18n/en/cosmic_settings.ftl:163]: Yeah I just noticed. I added -appearance to the one in Appearance. Let me know if it should be done differently.

Comment (git-f0x) [reply i18n/en/cosmic_settings.ftl:163]: Also should hot-corner be removed, or is it planned to be added at some point?

Comment (wash2) [reply i18n/en/cosmic_settings.ftl:163]: I think it is ok to leave it in for now

**Comment by git-f0x** [comment]
This might need some tiny code styling changes, since things aren't alphabetically arranged in some places.

**Comment by wash2** [comment]
> This might need some tiny code styling changes, since things aren't alphabetically arranged in some places.

Ya that would be good

**Comment by git-f0x** [comment]
Since the Desktop and Panel page was removed, this might create issues with Launcher searching "Panel" on existing installs, since the non-functional desktop entry for Desktop and Panel remains.

**Comment by wash2** [comment]
https://github.com/pop-os/cosmic-settings/blob/master/debian/install also needs to be updated to remove the old file and add the new ones.

**Comment by git-f0x** [comment]
It seems the icons for the subpages are larger in the designs, but not sure how to change that, since [this function](https://github.com/pop-os/cosmic-settings/blob/34d2b82376594e689ced2bcf8a23e7be0167bf06/page/src/lib.rs#L110) doesn't seem to take any size.

**Pull request merged**